src/main/java/core/basesyntax/Employee.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Employee {
```

Comment:
The class definition should not start with an empty line. Please remove the empty line before the class declaration.